front/app/containers/Admin/settings/customize/createHandler.ts

```diff
@@ -0,0 +1,55 @@
+import { get } from 'lodash-es';
+import { UploadFile, Multiloc } from 'typings';
+
+type TUploadType = 'logo' | 'header_bg';
+
+export function createAddUploadHandler(type: TUploadType, setState) {
```

Comment:
I didn't manage to figure out the typing for this one (and also the `setParentState` function used in the other sub-components). If you know how, do let me know, because I agree it would be better to have it typed